contracts/amm/MiltonInternal.sol

```diff
@@ -309,14 +309,26 @@ abstract contract MiltonInternal is
         return _iporOracle;
     }
 
+    function getIporOracle() external view returns (address) {
+        return address(_iporOracle);
+    }
+
     function _getMiltonStorage() internal view virtual returns (IMiltonStorage) {
         return _miltonStorage;
     }
 
```

Comment:
On top of smart contract external methods on bottom internal and private - try to not mix visibility in listed methods like here external internal external